test/test_sparse_prop.py

```diff
@@ -0,0 +1,229 @@
+# Owner(s): ["module: sparse"]
+#
+# Test to ensure sparsity information propagates properly into traced graph.
```

Comment:
Could you rename `test/test_sparse_prop.py` to `test/export/test_sparse.py` as the tests here are export specific?

Reply:
Ah, much better place for that indeed. Thanks.